### ISSUE TITLE
Python-only interface doesn't work on gpvms

### ISSUE THREAD

**Issue by chenel** [DUNE/dune-plot-style#33]
As reported by @laurajfields :

> After setting up the version of Python that the beamsim uses, I can't even setup a virtual enviornment:
```
-bash-4.2$ python3 -m venv my_env
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ModuleNotFoundError: No module named 'encodings'
Current thread 0x00007f045ff22740 (most recent call first):
Aborted
```
> If I use the version of python3 that comes installed by default on the gpvms, I can at least set up a virtual environment without error messages, but then stumble when installing dependencies:
Here is the last bit of the error message I get then:
```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -I/usr/include/freetype2 -I/tmp/pip-build-con69jwc/pillow -I/usr/include/fribidi -I/dune/app/users/ljf26/dune-plot-style/my_env/include -I/usr/local/include -I/usr/include -I/dune/app/users/ljf26/dune-plot-style/my_env/include -I/usr/include/python3.6m -c src/_imagingmorph.c -o build/temp.linux-x86_64-3.6/src/_imagingmorph.o
  src/_imagingmorph.c:14:20: fatal error: Python.h: No such file or directory
   #include "Python.h"
            ^
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
Command "/dune/app/users/ljf26/dune-plot-style/my_env/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-con69jwc/pillow/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-fteterzd-record/install-record.txt --single-version-externally-managed --compile --install-headers /dune/app/users/ljf26/dune-plot-style/my_env/include/site/python3.6/pillow" failed with error code 1 in /tmp/pip-build-con69jwc/pillow/
```

**Comment by olantwin** [comment]
This is for python 3.6?

**Comment by chenel** [comment]
I did some homework on this this morning.

* Python 3.6 is the default OS version on the GPVMs.
* Python 3.9 is what's set up via UPS whenever one drags in ROOT or other elements of the DUNE software stack.

I had no trouble creating a venv, or installing the dependencies, if I set up the newer version of Python3 (or of ROOT, which depends on it) beforehand, so I think we just need to update the installation instructions to be clear about this.

**Comment by olantwin** [comment]
I think in that case, it's probably reasonable to declare that we require python ≥3.9, and mention that this should normally be provided by UPS when using the DUNE stack.

**Comment by chenel** [comment]
Yeah.  I'm writing some updates into the README as we speak.

**Comment by chenel** [comment]
will be addressed by #34

**Comment by chenel** [comment]
Ok.  now that #34 is merged, I'm going to provisionally resolve this one.  (there may be some other environmental situations within DUNE workflows that we have to figure out how to address, but we'll do that as they come in...)
